bookserver/routers/assessment.py

```diff
@@ -143,3 +166,355 @@ async def get_history(request: Request, request_data: HistoryRequest):
     ]
 
     return make_json_response(detail=res)
+
+
+# Used by :ref:`compareAnswers`
+@router.get("/getaggregateresults")
```

Comment:
Docs would be helpful -- what's the purpose of this endpoint? The link is definitely nice, but doesn't appear to be pushed to the RS components repo yet.